tests/test_biosmaple_sqlite.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import os
+import pprint
+import sqlite3
+import unittest
+from sqlite3 import Error
+
+# MODEL_DIR, INPUT_DIR, OUTPUT_DIR
+from tests import INPUT_DIR
+
+"""Test the ability to read enumerated biosamples from the SQLite biosmaple database."""
+
+"""Run as follows to get see test-time printouts:"""
+
+"""python -m pytest -sv  tests/test_capitalization.py"""
+
+# INPUT_DIR comes from __init__.py
+PWD = os.path.dirname(os.path.realpath(__file__))
+TEST_DATA = os.path.join(INPUT_DIR, "test_sample_info.yaml")
+
+biosmaple_sqlite_file = "/Users/MAM/Documents/biosample_basex.db"
```

Comment:
Should remove hardcoded paths wherever possible. We should try and use the [tempfile](https://docs.python.org/3/library/tempfile.html) module wherever possible, to create these artifacts if need be so you can use them in your assert statements.